templates/bundles/DemosPlanCoreBundle/DemosPlanStatement/fragment_statement.html.twig

```diff
@@ -52,23 +52,28 @@
                 :init-priority-areas="JSON.parse('{{ selectedPriorityAreas|default([])|json_encode|e('js', 'utf-8') }}')"
                 init-fragment-text="{{ fragment.text|default|wysiwyg }}"
                 procedure-id="{{ procedure }}"
+                statement-text="{{ templateVars.statement.text|default|escape|wysiwyg(hasPermission('feature_obscure_text') ? 'dp-obscure' : []) }}"
                 inline-template>
                 <div>
                     <p class="u-mt-0_5">{{ "text.statement.fragment"|trans|wysiwyg }}</p>
 
                     {# statement text #}
                     <p class="weight--bold">
                         {{ "statement"|trans }}
-                        <dp-copy-paste-button
-                            button-label="statement.copy"
-                            class="float--right"
-                            copy-content="{{ templateVars.statement.text|default|escape|wysiwyg(hasPermission('feature_obscure_text') ? 'dp-obscure' : []) }}"
+                        <dp-button
+                            class="float--right btn--blank color-main u-pr-0 font-size-medium"
+                            color="tertiary"
```

Comment:
As there is no "tertiary" color in the `DpButton` component, i guess you just want to get it to render a non-existing class? That looks like hacking the component which sadly does not validate that prop yet.

Can we implement the "text" variant for DpButton? I'd be happy to contribute here.

Reply:
changed to subtle button variant in 808f2dcd5dbbf112be117b10e068b9f54202bd28